packages/inter-protocol/src/provisionPool.js

```diff
@@ -107,22 +115,43 @@ export const start = async (zcf, privateArgs, baggage) => {
     M.interface('ProvisionPool', {
       getMetrics: M.call().returns(M.remotable('MetricsSubscriber')),
       getPublicTopics: M.call().returns(TopicsRecordShape),
-      ...publicMixinAPI,
+      getSubscription: M.call().returns(M.remotable('StoredSubscription')),
+      getGovernedParams: M.call().returns(M.or(M.record(), M.promise())),
+      getElectorateSubscription: M.call().returns(SubscriberShape),
+      getInvitationAmount: M.call().returns(M.promise()),
     }),
     {
-      getMetrics() {
-        return provisionPoolKit.public.getPublicTopics().metrics.subscriber;
-      },
-      getPublicTopics() {
-        return provisionPoolKit.public.getPublicTopics();
-      },
-      ...publicMixin,
+      getMetrics: () =>
+        provisionPoolKit.public.getPublicTopics().metrics.subscriber,
+      getPublicTopics: () => provisionPoolKit.public.getPublicTopics(),
+      getSubscription: () => paramManager.getSubscription(),
+      getGovernedParams: () => paramManager.getParams(),
+      getElectorateSubscription: () => paramManager.getSubscription(),
+      getInvitationAmount: paramManager.getInvitationAmount,
+    },
+  );
+
+  /** @type {Record<string, (...any) => unknown>} */
+  const governedApis = {};
+
+  const creatorFacet = prepareExo(
+    baggage,
+    'governorFacet',
+    M.interface('governorFacet', GovernorFacetShape),
+    {
+      getParamMgrRetriever: () =>
+        Far('paramRetriever', { get: () => paramManager }),
+      getInvitation: name => paramManager.getInternalParamValue(name),
+      getLimitedCreatorFacet: () => provisionPoolKit.machine,
+      /** @type {() => GovernedApis} */
+      getGovernedApis: () => Far('governedAPIs', governedApis),
+      getGovernedApiNames: () => Object.keys({}),
```

Comment:
These aren't needed as long as no one is calling `voteOnApiInvocation()`. (no one is.)

Reply:
There's helper that can replace the whole creatorFacet.
```
  const creatorFacet = makeDurableGovernorFacet(baggage, () => provisionPoolKit.machine);
```

Reply:
Please correct me if I am wrong, but to reach this helper, we need to use the handleParamGovernance.
One of the main goals of this PR is to replace the use of handleParamGovernance with makeParamManagerFromTerm to allow the contract to override the governed parameters

Reply:
> Please correct me if I am wrong, but to reach this helper, we need to use the handleParamGovernance.
One of the main goals of this PR is to replace the use of handleParamGovernance with makeParamManagerFromTerm to allow the contract to override the governed parameters

That's all true. Can you find a way to extend `contractHelper.js` so it includes this case?  Does it help to notice that `handleParamGovernance` calls `makeParamManagerFromTerms()`? I think it might work to add an `overrides` param to `handleParamGovernance`, and pass it through.

Reply:
I was not aware that updating the governance package was a possibility.
This approach does indeed require a lot less changes to be introduces to the provisionPool contract.

Changes requested implemented on commits:
- [f36c592](https://github.com/Agoric/agoric-sdk/pull/10638/commits/f36c5929a002e6ca13f3ad4ae59301becc77aca5)
- [d3c95ae](https://github.com/Agoric/agoric-sdk/pull/10638/commits/d3c95ae5517b8b36c00e9b0c3884a90dca19d5b9)